tests/data/virtual-server-configmap-keys/configmap-validation-keys.yaml

```diff
@@ -11,4 +11,5 @@ data:
   max-fails: "3"
   keepalive: "32"
   proxy-protocol: "true"
-  redirect-to-https: "true"
\ No newline at end of file
+  redirect-to-https: "true"
+  upstream-zone-size: "0" # special value
```

Comment:
It doesn't look that in the tests we're checking that for Plus we generate zones with the default size for this case